src/main.js

```diff
@@ -0,0 +1,54 @@
+#!usr/bin/env node
+let loader = require('taskcluster-lib-loader');
+let App = require('taskcluster-lib-app');
+let monitor = require('taskcluster-lib-monitor');
+let debug = require('debug')('app:main');
+let config = require('typed-env-config');
+
+
+// Create component loader
+let load = loader({
+  cfg: {
+    requires: ['profile'],
+    setup: ({profile}) => config({profile}),
+  },
+
+  monitor: {
+    requires: ['process', 'profile', 'cfg'],
+    setup: ({process, profile, cfg}) => monitor({
+      project: 'taskcluster-queue',
+      credentials: cfg.taskcluster.credentials,
+      mock: cfg.monitor.mock,
+      process,
+    }),
+  },
+
+  server: {
+    requires: ['cfg', 'monitor'],
+    setup: ({cfg, monitor}) => {
+      let app = App({
```

Comment:
dustin just upgraded all the libraries, so to work differently, so maybe have a look a the version `10` of all libs...